tests/tracing/export/test_mlflow_exporter.py

```diff
@@ -0,0 +1,194 @@
+import time
+from threading import Thread
+from unittest.mock import MagicMock
+
+import pandas as pd
+from opentelemetry.sdk.trace import ReadableSpan
+
+from mlflow.tracing.export.mlflow import InMemoryTraceDataAggregator, MLflowSpanExporter
+from mlflow.tracing.types.model import Span, SpanContext, SpanType, Status, StatusCode
+from mlflow.tracing.types.wrapper import MLflowSpanWrapper
+
+
+def test_export():
+    trace_id = "trace_id"
+    otel_span_root = ReadableSpan(
+        name="test_span",
+        context=SpanContext(trace_id, "span_id_1"),
+        parent=None,
+        attributes={
+            "key1": "value1",
+        },
+        start_time=0,
+        end_time=4,
+    )
+
+    otel_span_child_1 = ReadableSpan(
+        name="test_span_child_1",
+        context=SpanContext(trace_id, "span_id_2"),
+        parent=otel_span_root.context,
+        attributes={
+            "key2": "value2",
+        },
+        start_time=1,
+        end_time=2,
+    )
+
+    otel_span_child_2 = ReadableSpan(
+        name="test_span_child_2",
+        context=SpanContext(trace_id, "span_id_3"),
+        parent=otel_span_root.context,
+        start_time=2,
+        end_time=3,
+    )
+
+    mock_client = MagicMock()
+    exporter = MLflowSpanExporter(mock_client)
+
+    # Export the first child span -> no client call
+    exporter.export([MLflowSpanWrapper(otel_span_child_1)])
+
+    assert mock_client.log_trace.call_count == 0
+    assert len(exporter._trace_aggregator._traces[trace_id].spans) == 1
+
+    # Export the second child span -> no client call
+    exporter.export([MLflowSpanWrapper(otel_span_child_2)])
+
+    assert mock_client.log_trace.call_count == 0
+    assert len(exporter._trace_aggregator._traces[trace_id].spans) == 2
+
+    # Export the root span -> client call
+    root_span = MLflowSpanWrapper(otel_span_root)
+    root_span.set_inputs({"input1": "very long input" * 100})
+    root_span.set_outputs({"output1": "very long output" * 100})
+    exporter.export([root_span])
+
+    assert mock_client.log_trace.call_count == 1
+    client_call_args = mock_client.log_trace.call_args[0][0]
+
+    # Trace info should inherit fields from the root span
+    trace_info = client_call_args.trace_info
+    assert trace_info.trace_id == trace_id
+    assert trace_info.name == "test_span"
+    assert trace_info.start_time == 0
+    assert trace_info.end_time == 4
+
+    # Inputs and outputs in TraceInfo should be serialized and truncated
+    assert trace_info.inputs.startswith('{"input1": "very long input')
+    assert len(trace_info.inputs) == 300
```

Comment:
Do you want to use the defined constant here to limit the amount of test updates needed when we need to change this based on feedback?

Reply:
Absolutely, will replace with a constant.